django_toolkit/fallbacks/circuit_breaker/rules.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import abc
+import logging
+
+logger = logging
+
+
+class Rule(object):
+    __metaclass__ = abc.ABCMeta
+
+    def __init__(
+        self,
+        max_failures,
+        failure_cache_key,
+    ):
+        self.failure_cache_key = failure_cache_key
+        self.max_failures = max_failures
+
+    @abc.abstractmethod
+    def should_open_circuit(self, total_failures, percentage_failures=None):
```

Comment:
`percentage_failures` doesn't need a default value, right? It will always be provided by the `CircuitBreaker`, even when it is not necessary.